presto-spi/src/main/java/io/prestosql/spi/security/AccessDeniedException.java

```diff
@@ -396,6 +396,11 @@ public static void denySetRole(String role)
         throw new AccessDeniedException(format("Cannot set role %s", role));
     }
 
+    public static void denyExecuteProcedure(String procedureName)
+    {
+        throw new AccessDeniedException("Cannot invoke procedure " + procedureName);
```

Comment:
Use `format` to be consistent with the others

Reply:
Maybe others should be fixed instead? Because: https://github.com/prestosql/presto#code-style
> Consider using String formatting ... Sometimes, if you only need to append something, consider using the + operator.

Reply:
@dain wants all of these to use format

Reply:
Maybe we should have a wiki page about Code Style? README section is too small to cover everything.
If you agree, I can start one.

Reply:
I completely agree. Discussions about code style belong there, not in code-review.
Although that specific example is quite clear for me. You can use both format or + in simple cases. This may be a simple case but format has been there already so it is the right choice here.